### PR TITLE
fixed #56 - created a staging and final database

### LINKED ISSUE #56: Create a staging and tranformed database
Right now everything is going into 1 database. We need this to be 2 databases. One for input flow docs. One for the conformed docs.

Input flow => staging db
Conformance Flow => tranformed db

### PR DESCRIPTION
**Pull request by paxtonhare**

### CODE REVIEW

File: data-hub/src/main/java/com/marklogic/hub/DataHub.java [hunk 1]
```diff
@@ -55,52 +61,91 @@
...
 import com.marklogic.mgmt.appservers.ServerManager;
+import com.marklogic.mgmt.databases.DatabaseManager;
+import com.marklogic.mgmt.restapis.RestApiManager;
+import com.marklogic.rest.util.Fragment;
 
 public class DataHub {
 
     static final private Logger LOGGER = LoggerFactory.getLogger(DataHub.class);
-
+    static final private String STAGING_NAME = "data-hub-in-a-box-STAGING";
```
Comment (dmcassel) [inline data-hub/src/main/java/com/marklogic/hub/DataHub.java]: Are these default names that can get overridden by configuration files?

File: quick-start/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-server.port=8080
+server.port=8081
 server.contextPath=/
 spring.thymeleaf.cache=true
 logging.level.com.marklogic.hub: DEBUG
 mlHost.default=localhost
-mlRestPort.default=8010
+mlStagingRestPort.default=8010
+mlFinalRestPort.default=8011
 mlUsername.default=admin
```
Comment (dmcassel) [inline quick-start/src/main/resources/application.properties]: Looks like this username is used to configure the hub. If it's also going to be used for application access, are there situations in which staging and final (different app servers) should use different credentials?

File: data-hub/src/main/java/com/marklogic/hub/DataHub.java (hunk 1 above)
Comment (paxtonhare) [inline data-hub/src/main/java/com/marklogic/hub/DataHub.java]: Not at this time.

File: quick-start/src/main/resources/application.properties (hunk 2 above)
Comment (paxtonhare) [inline quick-start/src/main/resources/application.properties]: We decided to keep it crazy simple. It's Quick Start and thus it's more of a demo of the hub.

File: data-hub/src/main/java/com/marklogic/hub/DataHub.java (hunk 1 above)
Comment (dmcassel) [inline data-hub/src/main/java/com/marklogic/hub/DataHub.java]: Okay. Probably worth making a note (or issue) to come to that.

**Pull request merged**